projects/packages/my-jetpack/_inc/components/product-cards-section/style.module.scss

```diff
@@ -58,6 +58,15 @@
 	}
 }
 
+.valueSectionHeading {
+	font-size: var(--font-body-extra-small);
+	color: var(--jp-gray-100);
+	font-weight: 500;
+	margin-bottom: calc(var(--spacing-base) + 2px);
+	line-height: var(--font-title-small);
+	text-align: center;
```

Comment:
Not sure the text-align is needed since the items are aligned with `flex-start`?